src/devices/video/mc6847.cpp

```diff
@@ -406,6 +432,9 @@ inline void mc6847_friend_device::next_scanline()
 		m_logical_scanline_zone = SCANLINE_ZONE_BOTTOM_BORDER;
 		enter_bottom_border();
 	}
+
+	// This helps to confirm synchronization between the screen device and the 6847
+	LOGMASKED(LOG_NEXTLINE, "mc6847_friend_device::next_scanline(): (vpos,hpos)=(%d,%d), m_physical_scanline='%d', m_logical_scanline_zone='%s', m_logical_scanline='%d'\n", (screen().vpos()), (screen().hpos()), m_physical_scanline, (scanline_zone_string((scanline_zone)m_logical_scanline_zone)), m_logical_scanline);
```

Comment:
I feel like the comment is redunant, at most it should be on same line of the `LOG_NEXTLINE` definition itself above.

Reply:
Makes sense.  I've removed the comment.